src/FormTaxonomies/Actions/UpdateFormTaxonomies.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Give\FormTaxonomies\Actions;
+
+use Give\DonationForms\Models\DonationForm;
+use WP_REST_Request;
+
+/**
+ * @unreleased
+ */
+class UpdateFormTaxonomies
```

Comment:
Can you write a test for this?

Reply:
Resolved, see [03fde79](https://github.com/impress-org/givewp/pull/7463/commits/03fde7956e5c01ef2f1c19604bf38c7763e9fa88).

Reply:
Thanks!